.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Pull Request
+about: Create a Pull Request to apply changes
+---
+
+
+**What is the problem I am trying to address**
+<!--Mention the issue you have been trying to solve -->
```

Comment:
I think for consistency with the other templates we should remove the html comment notation. `<!-- -->` And just keep the text.